### PR TITLE
feat: add functions to copy decoded data to clipboard in UI

### PR DESCRIPTION
**Pull request by xytian315**

### CODE REVIEW

File: assets/index.html [hunk 1]
```diff
@@ -76,12 +77,50 @@
...
+                  class="btn btn-default btn-sm"
+                  title="Copy header and payload to clipboard"
+                  data-bs-toggle="tooltip"
+                  data-bs-placement="bottom"
+                  data-bs-title="Copy header and payload to clipboard"
+                  onclick="copyToClipboard('jwtoutput', this)"
                 >
+                  <!-- bootstrap icon clipboard-->
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
```
Comment (dchinn1) [inline assets/index.html:101]: Can we add this as a file to our project rather than reference a webpage?

File: assets/index.html [hunk 2]
```diff
@@ -126,21 +165,104 @@
...
+                      title="Copy header to clipboard"
+                      data-bs-toggle="tooltip"
+                      data-bs-placement="bottom"
+                      data-bs-title="Copy header to clipboard"
+                      onclick="copyToClipboard('jwtoutputheader', this)"
+                    >
+                      <!-- bootstrap icon clipboard-->
+                      <svg
+                        id="jwtoutputheadersvgclipboard"
+                        xmlns="http://www.w3.org/2000/svg"
```
Comment (dchinn1) [inline assets/index.html:187]: same here

File: assets/index.html [hunk 3]
```diff
@@ -126,21 +165,104 @@
...
+                      class="btn btn-default btn-sm"
+                      title="Copy payload to clipboard"
+                      data-bs-toggle="tooltip"
+                      data-bs-placement="bottom"
+                      data-bs-title="Copy payload to clipboard"
+                      onclick="copyToClipboard('jwtoutputpayload', this)"
+                    >
+                      <!-- bootstrap icon clipboard-->
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
```
Comment (dchinn1) [inline assets/index.html:236]: and here

**Review by dchinn1 — COMMENTED** [review]
Do we want to leave in these links, or can we hard-add them to the repo?

File: assets/index.html (hunk 1 above)
Comment (xytian315) [reply assets/index.html:101]: Here is some documentation regarding svg https://developer.mozilla.org/en-US/docs/Web/SVG 

> Scalable Vector Graphics (SVG) is an XML-based markup language for describing two-dimensional based vector graphics.
As such, it's a text-based, open Web standard for describing images that can be rendered cleanly at any size...

https://icons.getbootstrap.com/#usage Embedded 
> Embed your icons within the HTML of your page (as opposed to an external image file).

**Comment by xytian315** [comment]
> Do we want to leave in these links, or can we hard-add them to the repo?

Replied in thread. It is svg. Are you asking to use img? There are some blogs to talk about this kind of topic https://www.growfox.co.uk/blog/5-reasons-you-should-be-using-svgs-over-pngs

I sort of prefer to use svg over img for our little project.

File: assets/index.html (hunk 1 above)
Comment (dchinn1) [reply assets/index.html:101]: This works without internet. No issue.

**Review by dchinn1 — APPROVED** [review]
lgtm!

**Pull request merged**